frontend/src/metabase/dashboard/containers/DashboardHeader.styled.jsx

```diff
@@ -5,8 +5,8 @@ import Button from "metabase/core/components/Button";
 
 export const DashboardHeaderActionDivider = styled.div`
   height: 1.25rem;
-  padding-left: 0.75rem;
-  margin-left: 0.75rem;
+  padding-left: 0.5rem;
```

Comment:
To match `QuestionActionsDivider`'s margin

https://github.com/metabase/metabase/pull/23967/files#diff-bf5deac9a0c26dd2003e0b7fd4682e4cca2fdedab92aba36f3b52f008edb5346R7-R8